File: 백준/Bronze/2588. 곱셈/곱셈.py
A=int(input())
B=int(input())

B_1 = B-(B//10)*10
B_100 = (B//100)
B_10 = (B-(B_100*100 + B_1))//10

result_1 =  A*B_1
result_2 = A*B_10
result_3 = A*B_100

print(result_1)
print(result_2)
print(result_3)
print(A*B)

File: 백준/Bronze/2588. 곱셈/곱셈.cs
using System;
public class Program
{

    public static void Main()
    {
        
        int A = Convert.ToInt32(Console.ReadLine());
        int B = Convert.ToInt32(Console.ReadLine());
        
        int B_1 = B - (B/10)*10;
        int B_100 = (B/100);
        int B_10 = (B - (B_100 * 100 + B_1))/10;

        int result_1 = A * B_1;
        int result_2 = A * B_10;
        int result_3 = A * B_100;

        Console.WriteLine(result_1);
        Console.WriteLine(result_2);
        Console.WriteLine(result_3);
        Console.WriteLine(A * B);


    }
}
